relay-event-normalization/src/transactions/processor.rs

```diff
@@ -25,19 +23,12 @@ pub struct TransactionNameConfig<'r> {
 #[derive(Debug, Default)]
 pub struct TransactionsProcessor<'r> {
     name_config: TransactionNameConfig<'r>,
-    timestamp_range: Option<Range<UnixTimestamp>>,
 }
 
 impl<'r> TransactionsProcessor<'r> {
     /// Creates a new `TransactionsProcessor` instance.
-    pub fn new(
-        name_config: TransactionNameConfig<'r>,
-        timestamp_range: Option<Range<UnixTimestamp>>,
```

Comment:
`timestamp_range` is used during validation, so not useful in the processor anymore.